Source/JavaScriptCore/wasm/WasmAirIRGenerator.cpp

```diff
@@ -3560,9 +3565,11 @@ auto AirIRGenerator::addArrayGet(uint32_t typeIndex, ExpressionType arrayref, Ex
     auto getValue = g64();
     // FIXME: Emit this inline.
     // https://bugs.webkit.org/show_bug.cgi?id=245405
-    emitCCall(&operationWasmArrayGet, getValue, instanceValue(), addConstant(Types::I32, typeIndex), arrayref, index);
+    emitCCall(&(arrayGetKind == GCOpType::ArrayGet ? operationWasmArrayGet :
```

Comment:
Can this be a variable?

Reply:
Actually, why can't this just be inline? It just adds some bit math and/or sign extend after, right?

Reply:
I've now inlined all of the signed/unsigned specific operations.